app/Http/Controllers/GetCourseController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Course;
+
+class GetCourseController extends Controller
+{
+    const PAGINATION = 14;
```

Comment:
ko nên để PAGINATION là const của controller, nên để các thông tin này trong config. Tương tự với các phần text trong code https://github.com/alexeymezenin/laravel-best-practices#use-config-and-language-files-constants-instead-of-text-in-the-code